src/index/hnsw/hnsw_config.h

```diff
@@ -15,11 +15,15 @@
 #include "knowhere/comp/index_param.h"
 #include "knowhere/config.h"
 
-// This is not a valid EF value for HNSW
-// This value is used to tell if HnswConfig.ef is coming from user or not
-const int64_t kDefaultHnswEfPlaceholder = -1;
-
 namespace knowhere {
+
+namespace {
+
+constexpr const CFG_INT::value_type kDefaultEfDivider = 16;
+constexpr const CFG_INT::value_type kDefaultRangeSearchEf = 16;
```

Comment:
can we use one "kDefaultEf" for both search and range search ?

Reply:
They don't have to be the same